webapp/channels/src/utils/text_formatting.tsx

```diff
@@ -618,6 +637,10 @@ export function escapeRegex(text?: string): string {
     return text?.replace(/[-/\\^$*+?.()|[\]{}]/g, '\\$&') || '';
 }
 
+export function escapeReplaceEspecialPatterns(text?: string): string {
```

Comment:
Nit: there's a bit of a Spanish twist here 🇪🇸 🥘 💃 

```suggestion
export function escapeReplaceSpecialPatterns(text?: string): string {
```